compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEControlCodeToNpu.cpp

```diff
@@ -285,6 +215,20 @@ struct HalfDmaCpyNdToNpuConverter final
         strides);
     if (failed(npuPushToQueueOp)) return failure();
     rewriter.replaceOp(op, *npuPushToQueueOp);
+
+    bool useNextBd = op.getUseNextBd();
+    if (useNextBd)
+      // Erase if not end of chain.
+      rewriter.eraseOp(*npuPushToQueueOp);
+    else {
```

Comment:
```suggestion
    bool useNextBd = op.getUseNextBd();
    if (useNextBd) {
      // Erase if not end of chain.
      rewriter.eraseOp(*npuPushToQueueOp);
    } else {
```
https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching